docs/release-notes/v12.2.2.md

```diff
@@ -0,0 +1,44 @@
+---
+id: v12.2.2
+title: Version 12.2.2 Release Notes
+sidebar_label: v12.2.2
+image: "img/release-note-link-preview.png"
+---
+
+_February 22th, 2024_
+
+## Console
+
+### New User events for the Webhook section
+
+User events have been added that allow a Company Owner to configure and trigger specific webhooks when changes to user access management occur.
+
+Listed below are the events that have been added with this release:
+* User added
+* User edited
+* User removed
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '500px'}}> 
+
+![IAM events webhook](./img/iam-events-webhook.png)
+
+  </div>
+</div>
+
+For more information regarding individual events, please [go to the documentation](/development_suite/webhooks-and-events/events.mdx#user-added).
+
+### Compare changes between a revision save and the previous one
+
+On Projects with the Enhanced Workflow, you can now make a comparison of the changes between a saved revision and the previous one. This can be done by simply going inside the revision History section – accessible from the related icon on the revision action popover – and clicking on the button on the side of each saved revision: a diff modal will show up, highlighting the changes between the selected revision save and the previous one.
```

Comment:
```suggestion
On Projects with the Enhanced Workflow, you can now make a comparison of the changes between a saved revision and the previous one. This can be done by simply going inside the revision History section – accessible from the related icon on the revision action popover – and clicking on the button on the side of each saved revision: a diff modal will show up, highlighting the changes between the selected revision save and the previous one.

```